src/main/kotlin/no/nav/syfo/api/PapirsykemeldingReglerApi.kt

```diff
@@ -0,0 +1,20 @@
+package no.nav.syfo.api
+
+import io.ktor.application.call
+import io.ktor.request.receive
+import io.ktor.response.respond
+import io.ktor.routing.Route
+import io.ktor.routing.post
+import no.nav.syfo.service.PapirsykemeldingRegelService
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+val log: Logger = LoggerFactory.getLogger("no.nav.syfo.smpapirregler")
```

Comment:
Kan gjerne legge til:
log.info("Got an request to validate rules")
og ein metric som teller hvor ofte regelen slår til slik:
RULE_HIT_STATUS_COUNTER.labels(validationResult.status.name).inc()

Reply:
yes, tenker det kommer etterhvert når man starter på implementeringen av reglene etc.

Reply:
ok